cli/src/datafeed.ts

```diff
@@ -0,0 +1,116 @@
+import { command, option, subcommands } from 'cmd-ts'
+import { insertHandler as adapterInsertHandler } from './adapter'
+import { insertHandler as aggregatorInsertHandler } from './aggregator'
+import { IDatafeedBulk, IDatafeedBulkInsertElement, ReadFile } from './cli-types'
+import {
+  contractConnectHandler,
+  contractInsertHandler,
+  functionInsertHandler,
+  organizationListHandler,
+  reporterInsertHandler as delegatorReporterInsertHandler
+} from './delegator'
+import { insertHandler as listenerInsertHandler } from './listener'
+import { insertHandler as reporterInsertHandler } from './reporter'
+import { isValidUrl, loadJsonFromUrl } from './utils'
+
+export function datafeedSub() {
+  // datafeed bulk-insert --source ${source}
+
+  // TODOs
+  // datafeed bulk-remove --source ${source}
+  // datafeed bulk-activate --source ${source}
+  // datafeed bulk-deactivate --source ${source}
+
+  const insert = command({
+    name: 'bulk-insert',
+    args: {
+      data: option({
+        type: ReadFile,
+        long: 'source'
+      })
+    },
+    handler: bulkInsertHandler()
+  })
+
+  return subcommands({
+    name: 'datafeed',
+    cmds: { insert }
+  })
+}
+
+export function bulkInsertHandler() {
+  async function wrapper({ data }: { data }) {
+    const bulkData = data as IDatafeedBulk
+
+    const chain = bulkData?.chain || 'localhost'
+    const service = bulkData?.service || 'DATA_FEED'
+    const organization = bulkData?.organization || 'bisonai'
+    const functionName = bulkData?.functionName || 'submit(uint256, int256)'
```

Comment:
I think there should be no space between parameters to produce the correct function signature.

```suggestion
    const functionName = bulkData?.functionName || 'submit(uint256,int256)'
```